### DISCUSSION TITLE
[OP Stack Deployment -- testnet] L2OutputOracle` nor `DisputeGameFactory` addresses were provided

### DISCUSSION THREAD

**Discussion by ub-tech** [ethereum-optimism/developers#537] [Deploying the OP Stack] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi -- on "start op-proposer" [step](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#start-op-proposer), getting the following error:

message="failed to setup: failed to init Driver: neither the `L2OutputOracle` nor `DisputeGameFactory` addresses were provided"

Thanks for any guidance.

**Comment by imtipi** [comment]
for L2OutputOracle,check this
https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-9145846
for DisputeGameFactory,check .deploy file and fill in `--game-factory-address=`

**Reply by ub-tech** [reply, marked as answer]
Thanks -- my fix was to grab the "L2OutputOracle" address from the .deploy file and add it manually to the "Start op-proposer" step.
